faq/templates/faq/inclusion_tags/faq_listing_item.html

```diff
@@ -0,0 +1,8 @@
+{% load ultracache_tags listing_tags i18n %}
```

Comment:
This template looks exactly the same as modelbase_listing_item. If that is indeed so it is not required because it will inherit.